.github/workflows/tests.yml

```diff
@@ -42,8 +42,6 @@ jobs:
     strategy:
       matrix:
         include:
-          - ruby-version: '2.6'
```

Comment:
@djezzzl this is the only version that fails to install sqlite3 for some reason...